hack/coverage.sh

```diff
@@ -13,9 +14,31 @@ declare -a EXCLUDE_FILES_FROM_COVERAGE=("nodemaintenance_controller_init.go")
 # delete coverage files (if present)
 find . -name ${COVERAGE_FILE} | xargs rm -f
 
-# run ginkgo with coverage result line
-${GINKGO} ${GINKGO_ARGS} ./pkg/ ./cmd/ | sed  '/coverage:.*$/d'
-GSTAT=${PIPESTATUS[0]}
+REPEAT_COUNT=0
+MAX_REPEAT_COUNT=10
+while [[ $REPEAT_COUNT -lt $MAX_REPEAT_COUNT ]]; do
+
+		# run ginkgo with coverage result line
+		${GINKGO} ${GINKGO_ARGS} ./pkg/ ./cmd/ 2>&1 | sed  '/coverage:.*$/d'  | tee msg
```

Comment:
Instead of using `tee` you might want just to use `grep`, this will be helpful in order to avoid creating new files on disks that later could appear as changes in git

Reply:
fixed.